### DISCUSSION TITLE
Flowforge in k3d

### DISCUSSION THREAD

**Discussion by EvdSvyat** [FlowFuse/flowfuse#1503] [Q&A] [answered]
Has anyone tried deploy flowforge in k3d? If yes, please tell me how, thanks!

**Comment by hardillb** [comment]
It should just be a case of deploying the helm chart (Assuming you have set the default Ingress controller).

If I get time at the weekend I'll give it a go

**Comment by hardillb** [comment, marked as answer]
Yep, works just fine with the helm chart

You will need to set up DNS to point to the LoadBalnance IP address assigned to the Traefik ingress controller which looks to default to 172.18.0.2

Hints on setting up DNS can be found here https://flowforge.com/docs/install/dns-setup/

**Comment by EvdSvyat** [comment]
Thank you!
